src/Catalyst.Core.Lib/CoreLibProvider.cs

```diff
@@ -152,7 +153,8 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<UploadFileTransferFactory>().As<IUploadFileTransferFactory>().SingleInstance();
 
             // Transaction validators
-            builder.RegisterType<TransactionValidator>().As<ITransactionValidator>().SingleInstance();
+            // builder.RegisterType<TransactionValidator>().As<ITransactionValidator>().SingleInstance();
+            builder.RegisterType<AlwaysTrueTransactionValidator>().As<ITransactionValidator>().SingleInstance();
```

Comment:
note the validation disabled until Stephen's changes are integrated

Reply:
@stxphxn